ni/measurementlink/discovery/v1/discovery_service.proto

```diff
@@ -176,4 +179,12 @@ message EnumerateComputeNodesRequest {
 message EnumerateComputeNodesResponse {
   // The list of compute nodes that have registered themselves in the current session.
   repeated ComputeNodeDescriptor compute_nodes = 1;
+}
+
+message ResolveServiceWithInformationResponse {
+  // The canonical location information for the service.
+  ServiceLocation service_location = 1;
+
+  // Url which provides descriptive information about the service.
+  string service_description_url = 2;
```

Comment:
This should be `ServiceDescriptor service_descriptor = 2;`. We have a need for capturing the annotations as well.